bodhi/tests/server/base.py

```diff
@@ -19,70 +19,198 @@
 import os
 import unittest
 
+from webtest import TestApp
 from sqlalchemy import event
-import requests
+import mock
 
-from bodhi.server import bugs, buildsys, log, models, initialize_db, Session, config
+from bodhi.server import bugs, buildsys, models, initialize_db, Session, config, main
 from bodhi.tests.server import create_update, populate
 
 
 original_config = config.config.copy()
+engine = None
+_app = None
 
-DB_PATH = 'sqlite://'
-DB_NAME = None
 
+def _configure_test_db(db_uri='sqlite:////var/tmp/bodhi-tests.sqlite'):
```

Comment:
I think this may fail to run in Koji. Should we make it a relative path to the source code instead so it will run on systems that restrict paths that can be written? Another option might be to make it a setting in development.ini that the spec file could adjust as needed, and it can default to this value. I could be wrong - maybe it'll be just fine on Koji as is…